nuntium/migrations/0003_denull_popolo_people.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('nuntium', '0003_add_parallel_popolo_data'),
+        ('instance', '0004_add_django_popolo_people'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='answer',
+            name='popolo_person',
+            field=models.ForeignKey(to='popolo.Person'),
+            preserve_default=True,
+        ),
+        migrations.AlterField(
```

Comment:
Oops, this `AlterField` and the next one are spurious - they were accidentally generated by having the old models file present after 1a0c81a4e78 and running makemigrations, I'll add a fixup to remove them.